src/MultiAgentReinforcementLearning/README.md

```diff
@@ -0,0 +1,17 @@
+# MultiAgentReinforcementLearning.jl
+
+This package provides multi agent reinforcement learning baselines as well as some further techniques.
+
+Basically, the following parts are included in this module
+
+- `MultiAgentManager`
+- `MADDPG`
+- `DIAL`
+- `CoordinationGraphs`
+- `MessageContainer` for `ReinforcementLearningTrajectory` Module
+- `wholeEpisodeSampler` for `ReinforcementLearningTrajectory` Module
```

Comment:
Should the capitalization here be consistent? E.g. ‘W’?

Reply:
DIAL stands for Differentiable Inter Agent Learning as proposed by [
Learning to Communicate with Deep Multi-Agent Reinforcement Learning (Jakob N. Foerster, ...)](https://arxiv.org/abs/1605.06676). Hence, all the great letters should be fine.
For `WholeEpisodeSampler` I will change that in the `README.md`.

Reply:
Changed in current commit of PR.